bindings/python/test_m68k.py

```diff
@@ -54,9 +54,9 @@ def print_read_write_regs(insn):
 def print_insn_detail(insn):
     if len(insn.operands) > 0:
         print("\top_count: %u" % (len(insn.operands)))
-        print("\tgroups_count: %u" % len(insn.groups))
 
     print_read_write_regs(insn)
+    print("\tgroups_count: %u" % len(insn.groups))
```

Comment:
~~Better use formatted strings for those cases (`f"\tgroups_count: {len(insn.groups)}"` and the like). Also below and above. Its easier for future editing IMHO.~~
Maybe doesn't matter that much. Since this should be replaced anyway by some proper testing code.

Reply:
(f-strings aren't supported in python 2 too)